.travis.yml

```diff
@@ -0,0 +1,10 @@
+---
+sudo: false
+language: ruby
+cache: bundler
+rvm:
+  - 2.5.1
```

Comment:
2.5.1. is not necessary if we test 2.5.5...

Reply:
oh I see